congrats/src/main/java/com/bedrock/congrats/ui/theme/Color.kt

```diff
@@ -13,3 +13,7 @@ val Pink80 = Color(0xFFEFB8C8)
 val Purple40 = Color(0xFF6650a4)
 val PurpleGrey40 = Color(0xFF625b71)
 val Pink40 = Color(0xFF7D5260)
+
+val PrimaryButton = Color(0xFF009EE3)
+val SecondaryButton = Color(0x26479AD1)
+val Transparent = Color(0x0000000)
```

Comment:
Let's think of a better naming for these colors, or at least try to keep this template for naming

This link may provide some ideaas https://developer.android.com/jetpack/compose/themes/material (check JetNews repo to check for color names)

```suggestion
val PrimaryButtonColor = Color(0xFF009EE3)
val SecondaryButtonColor = Color(0x26479AD1)
val TransparenButtonColor = Color(0x0000000)
```